dvc/command/plots.py

```diff
@@ -133,7 +133,9 @@ def add_parser(subparsers, parent_parser):
     plots_show_parser.add_argument(
         "targets",
         nargs="*",
-        help="Plots files to visualize. Shows all plots by default.",
+        help="Files to visualize (supports any file, "
+        "even when not found as `plots` in `dvc.yaml`). "
+        "Shows all plots by default.",
```

Comment:
This requires a small update in https://dvc.org/doc/command-reference/plots/show (top `usage` code block).

Reply:
p.s. Feel free to reuse iterative/dvc.org/pull/1809 for this.